sympy/sets/fancysets.py

```diff
@@ -1338,6 +1345,8 @@ def _contains(self, other):
         isTuple = isinstance(other, Tuple)
         if isTuple and len(other) != 2:
             raise ValueError('expecting Tuple of length 2')
+        if other is S.ComplexInfinity:
+            return False
```

Comment:
It seems that `zoo` is included in `Complexes` because `re(zoo)` and `im(zoo)` cannot be distinguished from proper numbers. `re` and `im` could be made to return something else, maybe `None` or `nan`.